### DISCUSSION TITLE
What zoom level does AO use?

### DISCUSSION THREAD

**Discussion by zippy77777** [kubilus1/autoortho#437] [General]
As the title please?

**Comment by Jeff2122** [comment]
I believe the setting for minimum is ZL12. I have tried upping that number to 14 but it does not do anything but slow it down and the scenery still looks the same, if it even completes loading.

**Reply by zippy77777** [reply]
So am I right in thinking that if you want ZL16, you go back to Ortho4XP?

**Reply by zodiac1214** [reply]
If you change it to 14, don't you have to remove already downloaded files? AO would just find it in cache/already downloaded which is your old zl12 images?

**Comment by Jeff2122** [comment]
Ortho4XP? Yes, I'm thinking that will do it. I fly helicopters sometimes and AO isn't cutting it for me flying low and slow. But AO v0.5.1 is working great for me at 3000' and above.
